### PR TITLE
Fix cscli inpsect json output

### PR DESCRIPTION
**Pull request by sbs2001**
Fixes https://github.com/crowdsecurity/crowdsec/issues/1139
Output now looks like:

```json
crowdsec-cli ) sudo ./crowdsec-cli collections  inspect crowdsecurity/sshd -o json                                                       fix_inspect_json ✭ ✱ ◼
{
 "type": "collections",
 "name": "crowdsecurity/sshd",
 "file_name": "sshd.yaml",
 "description": "sshd support : parser and brute-force detection",
 "author": "crowdsecurity",
 "path": "collections/crowdsecurity/sshd.yaml",
 "version": "0.2",
 "versions": {
  "0.1": {
   "digest": "21159aeb87529efcf1a5033f720413d5321a6451bab679a999f7f01a7aa972b3"
  },
  "0.2": {
   "digest": "72f6329808fafbb42da52cc6476a6e794d0a1ae5b3847e0060cf23593dd40352"
  }
 },
 "local_path": "/etc/crowdsec/collections/sshd.yaml",
 "local_version": "0.2",
 "local_hash": "72f6329808fafbb42da52cc6476a6e794d0a1ae5b3847e0060cf23593dd40352",
 "installed": true,
 "downloaded": true,
 "up_to_date": true,
 "tainted": true,
 "parsers": [
  "crowdsecurity/sshd-logs"
 ],
 "scenarios": [
  "crowdsecurity/ssh-bf",
  "crowdsecurity/ssh-slow-bf"
 ]
}⏎
```

### CODE REVIEW

File: cmd/crowdsec-cli/utils.go [hunk 1]
```diff
@@ -306,11 +306,25 @@ func InspectItem(name string, objecitemType string) {
 	if hubItem == nil {
 		log.Fatalf("unable to retrieve item.")
 	}
-	buff, err := yaml.Marshal(*hubItem)
-	if err != nil {
-		log.Fatalf("unable to marshal item : %s", err)
+	var b []byte
+	var err error
+	switch csConfig.Cscli.Output {
+	case "human":
```
Comment (buixor) [inline cmd/crowdsec-cli/utils.go:312]: human or raw should be same here

**Comment by buixor** [comment]
I think we might want to add some `omitempty` flags to structure fields to limit output noise

**Review by buixor — CHANGES_REQUESTED** [review]
minor change + add struct field tags

File: cmd/crowdsec-cli/utils.go (hunk 1 above)
Comment (sbs2001) [reply cmd/crowdsec-cli/utils.go]: fixed

**Review by buixor — APPROVED** [review]

**Pull request merged**